### PR TITLE
travis database setup

### PR DESCRIPTION
**Pull request by mossa-Sammer**
relates #17
database setup with travis done

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+before_script:
+  - 'psql -c "CREATE DATABASE db_travis;" -U postgres'
+  - 'psql -c "CREATE USER city WITH PASSWORD ''123'';" -U postgres'
+env:
+  global:
+    - PGPORT=5432
+    - SECRET=Mossa
```
Comment (AbdallahAmmar96) [inline .travis.yml]: Secret for what ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+before_script:
+  - 'psql -c "CREATE DATABASE db_travis;" -U postgres'
+  - 'psql -c "CREATE USER city WITH PASSWORD ''123'';" -U postgres'
```
Comment (AbdallahAmmar96) [inline .travis.yml]: Are you sure the user city is a good name ? 
And the same for the data base name

File: .travis.yml (hunk 1 above)
Comment (mossa-Sammer) [reply .travis.yml]: removed

File: .travis.yml (hunk 2 above)
Comment (mossa-Sammer) [reply .travis.yml]: changed

**Pull request merged**